woodwork/table_schema.py

```diff
@@ -552,7 +552,12 @@ def _filter_cols(self, include=None, exclude=None, col_names=False):
 
         return cols_to_return
 
-    def _get_subset_schema(self, subset_cols):
+    @property
+    def _get_subset_schema(self):
+        # removing this property might cause breaking changes in EvalML
+        return self.get_subset_schema
```

Comment:
Do we want to file an issue to track when this might be safe to remove?

Reply:
I think creating an issue makes sense - as far as I know once EvalML upgrades to this version and switches to using the public method, we can remove. Might actually want to create two issues - one in this repo to remove the private method and one in EvalML to upgrade WW and switch to using the public method.